### PR TITLE
New Repository and second attempt

### PR DESCRIPTION
**Pull request by FredsephT32**
My first repository had a lot of errors with the linters and the node.js and I decided to just create a new Repository

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -0,0 +1,45 @@
...
+  <title>Frederick's-Portfolio</title>
+  <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/4.7.0/css/font-awesome.min.css">
+  <link rel="stylesheet" href="styles.css">
+
+</head>
+<body>
+       <nav class="nav-bar">
+        <h1 class="Logo">My Logo</h1>
+        <img src="Images/Union.png" alt="Three-Lines">
+       </nav>
```
Comment (MeqdamAlqudah) [inline index.html]: - [ ] Please The container of the toolbar section must be a `header` tag

- [ ] Please the  hamburger button must be on the right 
image

Comment (MeqdamAlqudah) [inline index.html]: - [ ] Please the toolbar section should have the same margin-bottom right as  in the Figma design
image

File: index.html [hunk 2]
```diff
@@ -0,0 +1,45 @@
...
+          </div>
+
+          <div class="media-Images">
+               <img src="Images/Twitter-icon.svg"   alt="Twitter Icon">
+               <img src="Images/Linkedin-icon.svg"   alt="Linkedin Icon">
+               <img src="Images/Elegant-M-Icon.svg"   alt="Micro Icon">
+               <img src="Images/GitHub-Desktop-icon.svg" alt="Github Icon">
+               <img src="Images/Peace-Emoji-Icon.svg"  alt="Peace Icon">
+          </div>
+
```
Comment (MeqdamAlqudah) [inline index.html]: - [ ] Please must use a `ul` tag to group the list of icons, and `li` per icon; each icon has a link (`a` tag) 
as it is semantically better this in this way

File: index.html [hunk 3]
```diff
@@ -0,0 +1,45 @@
...
+        <div class="main-headline-txt">
+          <p>
+            I am a beginner student in the Microverse Program, this is my first week!
+            I'm learning right now how to implement flexbox properties in CSS. 
+            This is my first project using flexbox and setting up a mobile version skeleton
+          </p>
+
+          <div class="connect">
+               <h3 class="small-txt">LET'S CONNECT</h3>
+          </div>
```
Comment (MeqdamAlqudah) [inline index.html]: - [ ] Please ` let's connect` here should be a button not an h3 element

File: index.html [hunk 4]
```diff
@@ -0,0 +1,45 @@
...
+
+          <div class="media-Images">
+               <img src="Images/Twitter-icon.svg"   alt="Twitter Icon">
+               <img src="Images/Linkedin-icon.svg"   alt="Linkedin Icon">
+               <img src="Images/Elegant-M-Icon.svg"   alt="Micro Icon">
+               <img src="Images/GitHub-Desktop-icon.svg" alt="Github Icon">
+               <img src="Images/Peace-Emoji-Icon.svg"  alt="Peace Icon">
```
Comment (MeqdamAlqudah) [inline index.html]: - [ ] Please use the same margin as in Figma design fix this weird space around the page
image

**Review by MeqdamAlqudah — CHANGES_REQUESTED** [review]
Hi @FredsephT32 , 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

## Highlights


## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: index.html [hunk 5]
```diff
@@ -0,0 +1,58 @@
...
+  <link rel="stylesheet" href="styles.css">
+
+</head>
+<body>
+       <header class="header-bar">
+
+        <h1 class="Logo">My Logo</h1>
+        <a><img src="Images/Union.png" alt="Three-Lines"></a>
+        
+       </header>
```
Comment (mwanawabangona) [inline index.html]: - [ ] `optional`
- [ ] please make the logo same color as the figma template see screenshot below

your image
Screenshot from 2022-07-01 04-12-29

figma image
Screenshot from 2022-07-01 04-18-11

File: index.html [hunk 6]
```diff
@@ -0,0 +1,58 @@
...
+        <div class="main-headline-txt">
+          <p>
+            I am a beginner student in the Microverse Program, this is my first week!
+            I'm learning right now how to implement flexbox properties in CSS. 
+            This is my first project using flexbox and setting up a mobile version skeleton
+          </p>
+
+          <div class="connect">
+               <button class="small-txt">LET'S CONNECT</button>
+          </div>
```
Comment (mwanawabangona) [inline index.html:37]: - [ ] `optional`
- [ ] please make the button same color as the figma template see screenshot below

figma template
Screenshot from 2022-07-01 04-14-58

your template
Screenshot from 2022-07-01 04-12-43

**Review by mwanawabangona — APPROVED** [review]
## STATUS: APPROVED 🟢

## Hi @FredsephT32 👋,

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit:
Congratulations! 🎉




Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me @mwanawabangona in your question so I can receive the notification.

**Pull request merged**